docker/test/integration/runner/compose/docker_compose_kerberized_kafka.yml

```diff
@@ -50,7 +50,7 @@ services:
         - label:disable
 
   kafka_kerberos:
-    image: yandex/clickhouse-kerberos-kdc:${DOCKER_KERBEROS_KDC_TAG}
+    image: yandex/clickhouse-kerberos-kdc:${DOCKER_KERBEROS_KDC_TAG:-latest}
```

Comment:
Ok, but I don't understand the motivation of these changes.

Reply:
Just use the latest tag if don't provide anything